### PR TITLE
[Muhideen Mujeeb] - Sage - How to build a Celo Price Tracker browser extension using Vite and Celo Contractkit.

### PR DESCRIPTION
**Pull request by Mujhtech**
**Thanks for submitting this tutorial for Celo Sage**

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above. Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/2023-03-15-how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit/how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit.md [hunk 1]
```diff
@@ -0,0 +1,749 @@
...
+## Conclusion
+
+Congratulations, you have now learned how to build a browser extension base using vite and crxjs.
+
+## About the Author
+
+I am a Software Engineer, Tech Evangelist (Preaching the gospel of flutter & blockchain) also and Ex-GDSC Leads.
+
+## References
+
```
Comment (4undRaiser) [inline blog/2023-03-15-how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit/how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit.md:734]: These links should be embeded.

**Comment by 4undRaiser** [comment]
Also add a link to the github repository where the full project is hosted

**Comment by 4undRaiser** [comment]
Please also leave instructions on how to run and test the app after building. at the end of the tutorial.

File: blog/2023-03-15-how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit/how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit.md (hunk 1 above)
Comment (4undRaiser) [reply blog/2023-03-15-how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit/how-to-build-a-celo-price-tracker-browser-extension-using-vite-and-celo-contractkit.md:734]: The links in your references should some thing like this

[Celo Docs](https://docs.celo.org)

**Comment by 4undRaiser** [comment]
okay just fix the links and the github repo and we're good to go

**Comment by Mujhtech** [comment]
@4undRaiser fixed

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**